hub/saml-soap-proxy/src/test/java/uk/gov/ida/hub/samlsoapproxy/healthcheck/HealthCheckDataTest.java

```diff
@@ -3,52 +3,50 @@
 import org.junit.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
 
 public class HealthCheckDataTest {
 
     @Test
     public void shouldReturnEmptyHealthCheckDataWhenExtractedFromNullString() {
         HealthCheckData testData = HealthCheckData.extractFrom("test-id");
-        assertFalse(testData.getVersion().isPresent());
-        assertFalse(testData.getEidasEnabled().isPresent());
-        assertFalse(testData.getShouldSignWithSha1().isPresent());
+        assertThat(testData.getVersion().isPresent()).isFalse();
+        assertThat(testData.getEidasEnabled().isPresent()).isFalse();
+        assertThat(testData.getShouldSignWithSha1().isPresent()).isFalse();
     }
 
     @Test
     public void shouldReturnEmptyHealthCheckDataWhenExtractedFromEmptyString() {
         HealthCheckData testData = HealthCheckData.extractFrom("");
-        assertFalse(testData.getVersion().isPresent());
-        assertFalse(testData.getEidasEnabled().isPresent());
-        assertFalse(testData.getShouldSignWithSha1().isPresent());
+        assertThat(testData.getVersion().isPresent()).isFalse();
+        assertThat(testData.getEidasEnabled().isPresent()).isFalse();
+        assertThat(testData.getShouldSignWithSha1().isPresent()).isFalse();
     }
 
     @Test
     public void shouldReturnEmptyHealthCheckDataWhenExtractedFromSimpleIdString() {
         HealthCheckData testData = HealthCheckData.extractFrom("test-id");
         assertThat(testData.getVersion());
-        assertFalse(testData.getEidasEnabled().isPresent());
-        assertFalse(testData.getShouldSignWithSha1().isPresent());
+        assertThat(testData.getEidasEnabled().isPresent()).isFalse();
+        assertThat(testData.getShouldSignWithSha1().isPresent()).isFalse();
     }
 
     @Test
     public void shouldReturnHealthCheckDataWithVersionWhenExtractedFromStringThatContainsVersionData() {
         String expectedVersion = "1234";
         HealthCheckData testData = HealthCheckData.extractFrom("test-id-version-" + expectedVersion);
-        assertFalse(testData.getEidasEnabled().isPresent());
-        assertFalse(testData.getShouldSignWithSha1().isPresent());
-        assertTrue(testData.getVersion().get().contains(expectedVersion));
+        assertThat(testData.getEidasEnabled().isPresent()).isFalse();
+        assertThat(testData.getShouldSignWithSha1().isPresent()).isFalse();
+        assertThat(testData.getVersion().get().contains(expectedVersion));
```

Comment:
It _does_ actually add more detail because the first `contains` is from `java.lang.String` and it will only return true/false to the test framework but the second `contains` is from AssertJ and it will print a more detail message 👍